### PR TITLE
How do we validate a predictive model?

### PR DESCRIPTION
**Pull request by gottacatchenall**

### CODE REVIEW

**Comment by tpoisot** [comment]
I think this is still one question: what are the different approaches to validation and what do they tell us?

**Comment by gottacatchenall** [comment]
> I think this is still one question: what are the different approaches to validation and what do they tell us?

agreed that would make a better wording for the question, those questions i added are what i'm hoping to address in the answer---we'll see how it goes

**Comment by gottacatchenall** [comment]
@tpoisot @TanyaS08 this almost ready for review, but i'm looking for citation recommendations and any other comments before i attempt to merge

**Comment by tpoisot** [comment]
I suggest we merge after a review, and then when we integrate we can look at references. I like the fact that the current set of refs is technical and more hands-on discussions about model validation.

**Review by TanyaS08 — COMMENTED** [review]
Firstly, this is definitely on the better side of the scale of explaining model comparison/validation. Short and sweet but still informative. I'm assuming the plan is to expand on out of sample prediction? Also (out of curiosity) in one of the previous interactions hind casting was name dropped - is that a ship that has sailed/conversation for another day?

**Review by TanyaS08 — APPROVED** [review]
Happy to integrate as is if the rest of the review squad is

File: manuscript.md [hunk 1]
```diff
@@ -26,6 +26,21 @@
...
+One approach to avoid overfitting is using information criteria (e.g. AIC, BIC, MDL, the heavily maligned Bayes Factor), which are based around the heuristic that good models maximize the ratio of information provided by the model to the number of parameters it has. However, when the intended use-case of a model is prediction, _crossvalidation_ provides a better alternative for validating a model's predictive capacity. Crossvalidation methods divide the original dataset into two---one which is used to fit the model (called the _training_ set) and one used to validate its predictive accuracy on the data that is hasn't "seen" yet (called the _test_ set).
+This procedure is often repeated for different subdivisions of the dataset. One powerful approach is Leave-one-out-crossvalidation (LOOCV), which considers each data points uniquely as a test set, enabling sensitivity analysis. However, these methods are typically limited by the ensuing computation time requirements.
+
+- Out of sample prediction is ideal, but often not possible.
```
Comment (tpoisot) [inline manuscript.md:35]: Can we turn this into full sentences?

File: manuscript.md [hunk 2]
```diff
@@ -26,6 +26,21 @@
...
+- Bishop 2006, Gelman 2020 --- Bayesian Workflow
+
+
+(older stuff)
+
+Historically, ecologists have used information criteria (AIC, BIC, MDL, ...many more) to determine which of a set of competing models provide the “best” explanation for a set of data, based around the heuristic that good models maximize the ratio of information provided by the model to information encoded within the model. In the context of making good predictions, we are interested in validation to prevent overfitting and determine which model is most likely to make correct predictions in the future. For the purposes of prediction, crossvalidation is the better option as it directly tests predictive capacity on “unseen” data.
+Various forms of crossvalidation exist (k-fold, monte carlo, etc.)---often comes down to computation time limits.
```
Comment (tpoisot) [inline manuscript.md:42]: Can we develop this a little bit more?

**Review by tpoisot — CHANGES_REQUESTED** [review]

**Comment by TanyaS08** [comment]
@gottacatchenall is this one ready for review - last commit says still in progress os don't want to flag for review juuuuust in case!

**Comment by gottacatchenall** [comment]
> @gottacatchenall is this one ready for review - last commit says still in progress os don't want to flag for review juuuuust in case!

there may be some minor changes to phrasing i want to make but that can be done at the integration stage---good to merge if you are

**Pull request merged**